include/boost/beast/http/error.hpp

```diff
@@ -153,7 +153,16 @@ enum class error
         a new parser for each message. This can be easily done by
         storing the parser in an boost or std::optional container.
     */
-    stale_parser
+    stale_parser,
+
+    /** The body is shorter than claimed
+
+        This happens when writing to a stream from a file_body if
+        an end of file condition is discovered in the underlying
```

Comment:
"Discover" is non-sterile. Columbus "discovered" America, Curie "discovered" radioactivity. There's no discovery here, just the detection of a condition. Suggested:

```
/** The message body is shorter than expected.

    This error is returned by @ref file_body when an unexpected
    unexpected end-of-file condition is encountered while trying
    to read from the file.
*/
```